nixos/modules/module-list.nix

```diff
@@ -459,7 +459,15 @@
   ./services/x11/colord.nix
   ./services/x11/unclutter.nix
   ./services/x11/desktop-managers/default.nix
-  ./services/x11/display-managers/auto.nix
+  ./services/x11/desktop-managers/enlightenment.nix
+  ./services/x11/desktop-managers/gnome3.nix
+  ./services/x11/desktop-managers/kde4.nix
+  ./services/x11/desktop-managers/kde5.nix
+  ./services/x11/desktop-managers/kodi.nix
+  ./services/x11/desktop-managers/none.nix
+  ./services/x11/desktop-managers/xfce.nix
+  ./services/x11/desktop-managers/xterm.nix
+  #./services/x11/display-managers/auto.nix
```

Comment:
@groxxda Thanks for the suggestion, I added a commit that remove the file and the comment.